app/scripts/models/account.js

```diff
@@ -93,7 +91,7 @@ define([
           return self.createOAuthToken('profile:write')
             .then(function (accessToken) {
               self.set('accessToken', accessToken.get('token'));
-            }, function () {
+            }, function (err) {
```

Comment:
@shane-tomlinson nope, our ESLint rules only care about unused 'var's, not function args:
https://github.com/pdehaan/eslint-config-fxa/blob/master/index.js#L38